release-notes/3.0/releases.json

```diff
@@ -199,7 +199,9 @@
             "aspnetcore-runtime": {
                 "version": "3.0.0-preview-18579-0056",
                 "version-display": "3.0.0-preview-18579-0056",
-                "version-aspnetcoremodule": "13.0.18333.0",
+                "version-aspnetcoremodule": [
+                    "13.0.18333.0"
```

Comment:
I don't know that we would ever have a list of ancm versions for a given release. @shirhatti ?

Reply:
In all other new releases.json files the version-aspnetcoremodule field is specified as a list, and in some occasions multiple versions are indeed specified, e.g. [for release 2.1.6](https://github.com/dotnet/core/blob/master/release-notes/2.1/releases.json#L268-L272).

But while looking this up I also noticed that in the old releases.json this is not the case, so it might just be a mistake.

Reply:
I think this has been resolved, judging from f84493d626f21b42cebdaf3ea18e841f32ac550a